src/lib/cspNonce.js

```diff
@@ -0,0 +1,18 @@
+/* @flow */
+
+export type NormalizeNonceInputs = {|
```

Comment:
The `csp` types will improve after this PR is merged: https://github.com/paypal/paypal-sdk-client/pull/67